### ISSUE TITLE
Get the world name

### ISSUE THREAD

**Issue by JohnnyX071** [mircokroon/minecraft-world-downloader#370]
Hello, this is nice but how can I get the world name of the server I am in? Servers have many world's and I have no idea how they have named them

**Comment by JohnnyX071** [comment]
Yes but like what would I put in the world downloader if I want to download another world from a server

**Comment by mircokroon** [comment]
The download will save whatever chunks are sent to the client, so just load into whatever you want to download and it should be saved correctly.

**Comment by JohnnyX071** [comment]
But it will start downloading when I join, so I won't have time to change worlds
